### DISCUSSION TITLE
[Support]: Show detections not only alerts in home assistant media browser

### DISCUSSION THREAD

**Discussion by klonwar** [blakeblackshear/frigate#13456] [General Support] [answered]
### Describe the problem you are having

After updating to 0.14.1 I dont see detections in media browser dirs, only alerts
![image](https://github.com/user-attachments/assets/1884c756-996b-4926-9b0e-d39913e75fd5)

Any config option to change this behavior? I'd like to see all events in one place as it was previously. Right now I only see one way: make all events alerts which is not really good solution

### Version

0.14.1

### Frigate config file

```yaml
-
```

### Relevant Frigate log output

```shell
-
```

### Relevant go2rtc log output

```shell
-
```

### FFprobe output from your camera

```shell
-
```

### Operating system

Other Linux

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

-

**Comment by NickM-27** [comment, marked as answer]
currently this is working as expected based on the way review items are mapped to alerts, it will be improved in 0.15
